package.json

```diff
@@ -18,7 +18,7 @@
   "main": "app/index.js",
   "repository": "h5bp/generator-server-configs",
   "scripts": {
-    "test": "mocha --timeout 50000"
+    "test": "mocha --timeout 60000"
```

Comment:
No, it isn't too important (though it's 10 seconds of productivity gone).

Installing the dependencies should happen when the users uses the generator, though, why is installing the dependencies really needed to run the tests? Generally we don't do that at any yeoman generators.